trk/src/Components/ClimbItem.js

```diff
@@ -42,16 +45,31 @@ const ClimbItem = ({ climb, tapId }) => {
         navigation.navigate('Set', { climbData: climb });
     };
 
-    return (
-        <TouchableOpacity onPress={role === 'climber' ? navigateToDetail : navigateToSet}>
-            <ListItemContainer dotStyle={styles.climbDot}>
-                <Text style={styles.climbName}>{climb.name}</Text>
-                <View style={styles.setterDot}>
-                    {imageURL && <Image source={{ uri: imageURL }} style={{ width: '100%', height: '100%' }} />}
-                </View>
-            </ListItemContainer>
-        </TouchableOpacity>
-    );
+
+    if (fromHome) {
+        return (
+            <View>
+                <ListItemContainer dotStyle={styles.climbDot}>
+                    <Text style={styles.climbName}>{climb.name}</Text>
+                    <View style={styles.timerInfo}>
+                        <Text>{tapTimestamp}</Text>
+                    </View>
+                </ListItemContainer>
+            </View>
+        );
+    }
+    else {
+        return (
+            <TouchableOpacity onPress={role === 'climber' ? navigateToDetail : navigateToSet}>
+                <ListItemContainer dotStyle={styles.climbDot}>
+                    <Text style={styles.climbName}>{climb.name}</Text>
+                    <View style={styles.setterDot}>
+                        {imageURL && <Image source={{ uri: imageURL }} style={{ width: '100%', height: '100%' }} />}
+                    </View>
+                </ListItemContainer>
+            </TouchableOpacity>
+        );
+    }
```

Comment:
Above: new if/else statement to check against fromHome's boolean value. 
**only difference: display of timestamp instead of Image**

Reply:
that's great the only thing is I prop drilled a const called isFromHome to navigate from home to climbdetail and it's not a boolean its just a marker... don't know if it's a little confusing to have something similar but not exactly the same? we can probably keep both as is, just wanted to let you know

Reply:
A marker? What is a marker?

Reply:
<img width="824" alt="Screenshot 2023-12-13 at 2 16 22 pm" src="https://github.com/nagimonyc/trk-app/assets/66965648/cf6920b3-cdad-43fd-b66a-3c0c8fa8702a">

A boolean, you mean?